templates/ra_rdgw_autoscale_public_alb.template.cfn.json

```diff
@@ -210,6 +220,13 @@
             "Description": "A/B toggle that forces a change to a LaunchConfig property, triggering the AutoScale Update Policy",
             "Type": "String"
         },
+        "UpdateSchedule" :
+        {
+          "Default" : "",
+          "Description" : "(Optional) Time interval between auto stack updates",
+          "Type" : "String",
+          "ConstraintDescription" : "Must be in CloudWatch events schedule expression format (Cron or Rate)."
```

Comment:
I thought the ConstraintDescription is returned if the user enters a value that does not pass the AllowedPattern regex or AllowedValues list. Does this display somewhere if there is no actual constraint on the input? Perhaps consider a regex pattern?

Reply:
heh dunno.. added that at the last minute b/c I saw it on a reference template and looked like a good idea. Which time period pattern do you want me to enforce? Currently the custom resource will support both Rate and Cron: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html leaning towards rate as its a little more user friendly

Reply:
You could do both... `^$|...rate_regex...|...cron_regex...`